extensions/amp-analytics/0.1/variables.js

```diff
@@ -297,6 +299,26 @@ export class VariableService {
     });
   }
 
+  /**
+   * @param {Array<string>} array
+   * @param {!ExpansionOptions} options
+   * @return {Array<string>}
+   */
+  expandArray_(array, options) {
+    for (let i = 0; i < array.length; i++) {
+      // Treat each value as a template and expand
+      array[i] = this.expandTemplateSync(
```

Comment:
oh, I meant to extract this method to share with L280

Reply:
oops, got it